src/Esfa.Vacancy.Manage.Api/Controllers/CreateApprenticeshipController.cs

```diff
@@ -17,6 +17,25 @@ public CreateApprenticeshipController(CreateApprenticeshipOrchestrator orchestra
         }
         /// <summary>
         /// Creates the specified apprenticeship vacancy.
+        /// 
+        /// #### Error codes ####
+        /// 
+        /// The following error codes may be returned when calling this operation if any of the vacancy values 
+        /// specified fail validation:
+        /// 
+        /// | Error code  | Explanation                                                                      |
+        /// | ----------- | -------------------------------------------------------------------------------- |
+        /// | 31000       | Title was not specified                                                          |
+        /// | 31001       | Title must be no more than 100 characters                                        |
```

Comment:
Would 'Title cannot be longer than 100 characters' not read better?

Reply:
Yes it would.  Have updated.